google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineExecutor.java

```diff
@@ -23,14 +23,14 @@
 import com.intellij.openapi.vcs.impl.CancellableRunnable;
 
 /**
- * Runner of {@link AppEngineTask}'s.
+ * Executor of {@link AppEngineTask}'s.
  */
-public class AppEngineRunner implements CancellableRunnable {
+public class AppEngineExecutor implements CancellableRunnable {
```

Comment:
@etanshaul 

I'm noticing that this is a 'generic' executor but calls APP_ENGINE_DEPLOY_CANCEL ping on cancel.